packages/mui-material/src/Select/Select.test.js

```diff
@@ -898,6 +896,12 @@ describe('<Select />', () => {
         expect(errors).to.have.length(1);
         expect(errors[0].toString()).to.include('MUI: The `value` prop must be an array');
       });
+
+      it('should not crash with `value` passed', () => {
+        expect(() => {
+          render(<Select multiple value={['foo']} />);
+        }).not.to.throw();
+      });
```

Comment:
We are already testing the same test case above, no? It's safe to remove this one.

Reply:
Yes, it's a duplicate. My bad :p